### ISSUE TITLE
Content Update Request for Debt Portal from Debt Resolution team

### ISSUE THREAD

**Issue by mgleo** [department-of-veterans-affairs/va.gov-team#78039]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: `Debt Resolution team`
- **OCTO product owner:** `Denise Coveyduc`
- **Product name:** `Debt Portal`
- **Product manager:**  `Tom Davis`
- **Slack channel:** [#debt-resolution](https://join.slack.com/share/enQtNjc3NzE5NDY4Mzk1Ny0xZjcwN2Y1Zjg0OTA5MTRmNTkwMzU0MTU5N2Q5YTllZTUwNTIyOWJhNmQ1ZGFhMjVlZmZhODAyZmRjMjcxNjlk)
- **Dedicated content writer on your team (if you have one):** `Insert name and/or GH handle`
- **Dedicated a11y specialist on your team (if you have one):** `Insert name and/or GH handle`
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [x] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

**[Debt Portal UnAuth and Auth Landing pages](https://staging.va.gov/manage-va-debt/):**

- **_"Manage your VA debt"_** link should be an [action link](https://design.va.gov/components/link/action) on both Unauth and Auth pages

- Remove **_"Who can sign in to manage debt online"_** and **_"What you can do when you sign in"_** sections from Auth page only. Users will be signed in already if they are viewing this page.

- On UnAuth page under **_"What you can do when you sign in"_**. Update the bullet point **_"Download billing statements, debt notices, and collection letters"_** to read **_"Download copay billing statements"_**. 
This change is needed as Debt Letters are no longer up on the Debt Portal.

- We would also like to open a discussion on whether an Auth landing page is necessary on the Debt Portal. We are receiving feedback that users would prefer fewer clicks to get to information on the Debt Portal. The information on the auth landing page is also information that is available on the Debt Portal as well (repeating information).

<kbd><img width="717" alt="Screenshot 2024-03-11 at 12 24 34 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/5553542/529dc238-0194-4e36-b7a0-c728fee4200e"></kbd>

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

As soon as possible

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact: Debt Portal](https://staging.va.gov/manage-va-debt/) 
- [Topline summary from recent research study](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/research/exploratory-2024/topline-summary.md)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
- [x] Update 1 sentence on Manage your VA debt page (Laura) http://preview-prod.vfs.va.gov/preview?nodeId=6521
- [ ] Review and publish (Lily)
```

```[tasklist]
### CAIA IA tasks
```

**Comment by laurwill** [comment]
Hi @mgleo , responses to the 4 asks below! Let me know if you have questions or want to chat about any of this.

1. "Manage your VA debt" link should be an [action link](https://design.va.gov/components/link/action) on both Unauth and Auth pages
- I think there was a bug with sign-in widgets recently that made them look like text links — it's now showing correctly on the unauth page as a sign-in widget. (The action link should only appear on the auth version of the page, after the user has signed in.)
<img width="589" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/104158104/9927f53d-0a8f-4b8d-8902-b60932ee875f">

2. Remove "Who can sign in to manage debt online" and "What you can do when you sign in" sections from Auth page only. Users will be signed in already if they are viewing this page.
- Sitewide CAIA manages only the unauth content that lives in the Drupal CMS — you'll need to coordinate with the product team that owns the debt portal for changes to the auth content. 

3. On UnAuth page under "What you can do when you sign in". Update the bullet point "Download billing statements, debt notices, and collection letters" to read "Download copay billing statements".
This change is needed as Debt Letters are no longer up on the Debt Portal.
- No problem, we can make that accuracy change on the unauth page.

4. We would also like to open a discussion on whether an Unauth landing page is necessary on the Debt Portal. We are receiving feedback that users would prefer fewer clicks to get to information on the Debt Portal. The information on the Unauth landing page is also information that is available on the Debt Portal as well (repeating information).
- Thanks for raising this! This may be a larger conversation about the different entry points and user flows, but here's a few initial thoughts and a question: 
          - Users need to sign in to use the debt portal — so if we remove the unauth page with the sign-in widget, that would remove the entry point to the tool for unauth users. We provide unauthenticated landing pages for all VA online tools (for example, [Rx refill](https://www.va.gov/health-care/refill-track-prescriptions/)), so that users who either don't yet have sign-in accounts or aren't currently signed in can learn what the tool does and decide if they want to authenticate to use it. 
          - When you say "prefer fewer clicks to get to information on the debt portal," does that mean they want to get into the actual debt portal tool more quickly to check the amount of their debt? Or they want to find information about the debt portal tool more quickly? (I took a quick look at the linked research study, but didn't see a key finding related to this.)

**Comment by mgleo** [comment]
Hi @laurwill,
1. I saw the thread about this on Slack and looks like it may be taken care of soon [#17560](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17560)
2. Thanks for this information, I'll ask the FE team about this.
3. Thank you for making this update!
4. I misspoke and meant the Auth page, sorry for the confusion. I'll update the original ticket to change unauth to auth. Yes, participants would want to review how much they owe, some would want to dive deeper into the bill, and then quickly pay. All of this can be done on the Debt Portal Summary page: https://staging.va.gov/manage-va-debt/summary

**Comment by kristinoletmuskat** [comment]
Hey ya'll, for #4 above, we confirmed that we can route authenticated users directly to the manage debt tool without first displaying the authenticated landing page. Not all tools qualify to do this, but the manage debt tool has all the relevant information from the auth landing page, so users wouldn't be missing anythng from that page. We’re already doing this for the claim status tool, request COE (housing), and direct deposit tools. I'm not sure how those teams implemented that change, so feel free to reach out to them to learn more.


![Screenshot 2024-03-27 at 10 13 23 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/4e6e6a36-0b24-41cd-aedc-7538cf61928f)

![Screenshot 2024-03-27 at 10 15 02 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/bfe88f56-dae8-4b1a-b04a-b80988f8093a)

**Comment by mgleo** [comment]
Thanks @kristinoletmuskat, this is helpful.
cc @OptionSelect

**Comment by laurwill** [comment]
Also confirming we removed "debt notices and collection letters" from the list on the unauth page of what's available in the tool. I think that wraps up all the CAIA work in this ticket, right @mgleo ? Okay to close this out?

**Comment by mgleo** [comment]
Yes, ok to close, thank you!
